lib/content_script/init.js

```diff
@@ -15,7 +15,7 @@ v8Util.setHiddenValue(global, 'ipc-internal', v8Util.getHiddenValue(isolatedWorl
 // The process object created by webpack is not an event emitter, fix it so
 // the API is more compatible with non-sandboxed renderers.
 for (const prop of Object.keys(EventEmitter.prototype)) {
-  if (process.hasOwnProperty(prop)) {
+  if (Object.prototype.hasOwnProperty.call(process, prop)) {
```

Comment:
This could perhaps be better done with `Object.getOwnPropertyNames`?

Reply:
They are not functionally equivalent

![image](https://user-images.githubusercontent.com/6634592/77125474-13ed8200-6a03-11ea-8242-e95170e619d1.png)

Reply:
Ah... because `constructor` is included in `getOwnPropertyNames`. ok.